programs/drift/src/state/perp_market.rs

```diff
@@ -588,8 +588,8 @@ pub struct AMM {
     pub oracle_source: OracleSource,
     /// tracks whether the oracle was considered valid at the last AMM update
     pub last_oracle_valid: bool,
-
-    pub padding: [u8; 48],
+    pub target_base_asset_amount_per_lp: i32,
```

Comment:
should add comment here and on jit_intensity explaining why it is between 0 and 200